ingestr/src/attio/helpers.py

```diff
@@ -0,0 +1,56 @@
+import pendulum
+
+from ..utility import create_client
+
+base_url = "https://api.attio.com/v2"
+
+
+class AttioClient:
+    def __init__(self, api_key: str):
+        self.api_key = api_key
+        self.headers = {
+            "Accept": "application/json",
+            "Authorization": f"Bearer {self.api_key}",
+        }
+        self.client = create_client()
+
+    def fetch_data(self, url: str, method: str, limit: int = 1000, params=None):
+        if params is None:
+            params = {}
+        offset = 0
+        while True:
+            query_params = {**params, "limit": limit, "offset": offset}
+            if method == "get":
+                response = self.client.get(
+                    url, headers=self.headers, params=query_params
+                )
+            else:
+                response = self.client.post(
+                    url, headers=self.headers, params=query_params
+                )
+
+            if response.status_code != 200:
+                raise Exception(f"HTTP {response.status_code} error: {response.text}")
+
+            response_data = response.json()
+            if "data" not in response_data:
+                raise Exception(
+                    "Attio API returned a response without the expected data"
+                )
+
+            data = response_data["data"]
+
+            for item in data:
+                flat_item = flatten_item(item)
+                yield flat_item
+
+            if len(data) < limit:
+                break
+            offset += limit
+
+
+def flatten_item(item: dict) -> dict:
```

Comment:
I think we should flatten `values` as well if they exist

Reply:
Sabri and I decided to flat id only